resume-buddy/src/main/java/com/google/sps/data/Reviewee.java

```diff
@@ -10,7 +10,7 @@
   private final String career;
   private final Degree degreePref;
   private final NumYears numYearsPref;
-  private final String resumeURL;
+  private final String resumeBlobkey;
```

Comment:
Nit: `resumeBlobKey` because BlobKey is in camel case.

Reply:
Okay! I wasn't sure if blobkey was considered 1 word or 2